<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>records_add_window</class>
 <widget class="QDialog" name="records_add_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_fname"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_sname"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_thname"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Услуга</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_service"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDateEdit" name="dateEdit_date">
     <property name="maximumDate">
      <date>
       <year>2023</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2022</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="date">
      <date>
       <year>2022</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Время начала</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QTimeEdit" name="timeEdit_start">
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="minimumTime">
      <time>
       <hour>9</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="time">
      <time>
       <hour>9</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Время окончания</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTimeEdit" name="timeEdit_end">
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="minimumTime">
      <time>
       <hour>9</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="time">
      <time>
       <hour>9</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBox_status">
     <item>
      <property name="text">
       <string>Свободно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Занято</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>records_add_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>records_add_window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
